CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Lawnchair Contributing Guidelines
+
+<picture>
+    <!-- Avoid image being clickable with slight workaround --->
+    <source media="(prefers-color-scheme: dark)" srcset="docs/lawnchair-round.webp" width="100">
+    <img alt="" src="docs/lawnchair-round.webp" width="100">
+</picture>
+
+First and foremost, welcome to the **Lawnchair** Contributing Guidelines!
+This document will contain everything you'll need to contribute to the **Lawnchair** project.
+
+## 👋 Before you start!
+
+We would like you to be familiar with the tools we are using.
+The tool(s) you will be using depend on the type of contribution you are going to make.
+
+### 💫 Bug report & feature requests
+
+For bug reports, please describe the bug in detail to the best of your ability
+and steps on how to reproduce it. If applicable, attach log files.
+
+For feature requests, please describe the feature you'd like to see added to Lawnchair.
+If applicable, attach log files.
+
+Please make sure to be civil during discussion of your contribution (refer to the [Code of Conduct](CODE_OF_CONDUCT.md))
+
+### 🌐 Translation
+
+For translation, you only need to know one tool, [Crowdin](https://lawnchair.crowdin.com),
+which allows us to collaborate with multiple translators & proofreaders.
+
+-   When using quotation marks, insert the symbols specific to the target language, as listed in this [this summary table](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table);
+-   Lawnchair uses title case for some English UI text while other languages do not use title case; opt for sentence case instead;
+-   Some English terminology may have not have commonly-used equivalents in other languages. In such cases, use short descriptive phrases—for example, the equivalent of _bottom row_ for _dock_;
+-   Some languages (e.g. French) have variations of words depending if it's masculine or feminine (gender-specific); we recommend opting for gender-neutral words instead.
+
+### 🧑‍💻 Code
+
+> [!NOTE]
+> For Lawnchair 9 to 13, see the branches with the `9-` to `13-` prefixes respectively.
+
+For code, it's highly recommended that you use an IDE (e.g. [Android Studio](https://developer.android.com/studio)),
```

Comment:
```suggestion
For code, it's highly recommended that you use [Android Studio](https://developer.android.com/studio),
```
Android Studio is the preferred IDE and most other features won't work in others.